src/Transients.cc

```diff
@@ -25,7 +25,12 @@
 #include <limits>
 
 /// shared memory segment path to use for Transients map
-static const SBuf MapLabel("transients_map");
+inline static const auto &
```

Comment:
I assume this was done as a side effect of @rousskov request to match/copy PR #1745 review changes.

AIUI, The reasons for using `inline static` pertain specifically to the logic being in a shared header, they become counter-productive when the logic is defined as a `static` function within a unit (`.cc` file).

```suggestion
static const auto &
```

Reply:
Fine by me; the compiler can inline if it chooses to